docs/plugins/mosaic.rst

```diff
@@ -0,0 +1,57 @@
+Mosaic Plugin
+=====================
+
+The ``mosaic`` plugin generates a montage of a mosiac from cover art.
+
+To use the ``mosaic`` plugin, first enable it in your configuration (see
+:ref:`using-plugins`). Then, install the `Pillow`_ library by typing::
+
+    pip install Pillow
```

Comment:
Looks like this plugin also uses a library called `parse`, so you probably want to mention that here. (Or just use regular expressions instead.)